chisel-wrapper

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+# USAGE:
+#
+#   ./<script> [ chisel-cut-OPTIONS ] <slices..>
+#
+# Installs the slices, using
+#
+#   chisel cut "$@"
+#
+# Generates a status file similar to /var/lib/dpkg/status
+# at the CHISEL_DPKG_STATUS_FILE path.
+#
+# env vars:
+#   CHISEL_BIN                (optional) the path of chisel binary
+#   CHISEL_DPKG_STATUS_FILE   (required) the path where the dpkg status file should be generated
+
+set -eu
+
+CHISEL_BIN="${CHISEL_BIN:-"chisel"}"
+CHISEL_CACHE_DIR="$(mktemp -d)"
+
+cleanup() {
+	rm -rf "$CHISEL_CACHE_DIR"
+}
+trap cleanup EXIT
+
+print_error() {
+	echo "Error:" "$@" >> /dev/stderr
+}
+
+install_slices() {
+	XDG_CACHE_HOME="$CHISEL_CACHE_DIR" $CHISEL_BIN cut "$@"
+}
+
+prepare_dpkg_status() {
+	dir="$CHISEL_CACHE_DIR/chisel/sha256"
+	if [ ! -d "$dir" ]; then
+		print_error "could not find the chisel cache at ${dir}"
+		exit 1
+	fi
+
+	if [ -f "$CHISEL_DPKG_STATUS_FILE" ]; then
+		rm -f "$CHISEL_DPKG_STATUS_FILE"
+	fi
+
+	for f in "$dir"/*; do
+		is_deb="$(file "$f" | grep "Debian binary package" | cat)"
+		if [ -z "$is_deb" ]; then
+			continue
+		fi
+		dpkg-deb -f "$f" >> "$CHISEL_DPKG_STATUS_FILE"
+		echo "" >> "$CHISEL_DPKG_STATUS_FILE"
+	done
+}
+
+if [ -z "$CHISEL_DPKG_STATUS_FILE" ] ; then
```

Comment:
Since you have `set -u`, this will fail with variable unset error instead of going down into the if block. You can do either 

```
if [ -z "${CHISEL_DPKG_STATUS_FILE:-}" ]; then
```
or use Bashism
```
if [[ ! -v CHISEL_DPKG_STATUS_FILE ]]; then
```